### PR TITLE
DDF-3660 Update CDM permissions and docs

### PR DESCRIPTION
**Pull request by peterhuffer**
#### What does this PR do?
- Update CDM permissions. 
- Update docs

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@emmberk @bakejeyner @jefchng 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams

#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@clockard
@vinamartin

#### How should this be tested? (List steps with links to updated documentation)
Setup Move, Delete, and In Place configurations of CDM on directories outside of the user home directory according to the docs. Verify for each method that the products ingested are downloadable via Intrigue.

#### What are the relevant tickets?
[DDF-3660](https://codice.atlassian.net/browse/DDF-3660)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_configuring/content-directory-monitor.adoc [hunk 1]
```diff
@@ -19,7 +19,7 @@ The Content Directory Monitor is installed by default with a standard installati
 
 [TIP]
 ====
-If the CDM being configured is WebDav in place, or the directory being monitored is inside ${home_directory}, then adding permissions is not required and this section can be skipped.
+If the CDM being configured is WebDav in place then adding permissions is not required and this section can be skipped.
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/content/_configuring/content-directory-monitor.adoc]: ❓ Is this for `WebDav` or `In Place` then adding permissions isn't required?

Comment (peterhuffer) [reply distribution/docs/src/main/resources/content/_configuring/content-directory-monitor.adoc]: WebDav, but WebDav is in place.

**Comment by peterhuffer** [comment]
build now

**Comment by clockard** [comment]
Manually tested. Looks good! :shipit:

**Pull request merged**